README.md

```diff
@@ -14,21 +14,21 @@ Then follow the instructions below to set up `calyx` and `odgi`.
 ### Installing Dependencies
 
 #### Calyx
-Follow these [instructions](https://docs.calyxir.org/) to install calyx. You will need to configure the calyx driver, `fud`. We recommend using the native calyx interpreter, so once `fud` is set up, run
+Follow these [instructions](https://docs.calyxir.org/) to install calyx. You will need to complete the Installing the Commandline Driver section, but can skip Running the Core Tests. We recommend using the native calyx interpreter, so once `fud` is set up, run
 ```
-fud config stages.interpreter $ABSOLUTE_PATH_TO_CALYX_REPO/target/debug/interp
+fud config stages.interpreter <full path to Calyx repository>/target/debug/interp
 ```
-where `$ABSOLUTE_PATH_TO_CALYX_REPO` is the absolute path to the root directory. For example, if you downloaded calyx in `/Users/username/project`, you would run `fud config stages.interpreter /Users/username/project/target/debug/interp`.
+where `<full path to Calyx repo` is the absolute path to the root directory. For example, if you downloaded calyx in `/Users/username/project`, you would run `fud config stages.interpreter /Users/username/project/calyx/target/debug/interp`.
 
 #### Odgi
 
-You will need to install the python bindings for [odgi]. Instructions for installing odgi can be found [here](https://odgi.readthedocs.io/en/latest/rst/installation.html). If you compile odgi from its source, you will need to [edit your python path](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) to use the python bindings.
+You will need to install the python bindings for [odgi]. Instructions for installing odgi can be found [here](https://odgi.readthedocs.io/en/latest/rst/installation.html). Installing odgi via `bioconda` seems to be the most straightforward option. If you instead compile odgi from its source, you will need to [edit your python path](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) to use the python bindings. 
 
-To verify that you have the python bindings, open up a python shell and try `import odgi`. If this doesn't work, you can try a different method of installation, or you can download the `.so` files from [bioconda](https://anaconda.org/bioconda/odgi/files) for the version of python you are running (`python3 --version`) and add them to your `PYTHONPATH` instead.
+To verify that the python bindings are working, open up a python shell and try `import odgi`. If this doesn't work, you can also download the `.so` files from [bioconda](https://anaconda.org/bioconda/odgi/files) for the version of python you are running and add them to your `PYTHONPATH`. For example, if `python --version` is 3.7, fetch `odgi...py37....tar.bz2`. You will have to extract the `.so` files by unzipping the `.tar.bz2` file.
```

Comment:
I think this is getting hair enough that we need a little list, or even a section, to cover it.

Reply:
1. Check Python3 version. Let's say it is 3.9.12.
2. Make `odgi` directory.
3. Download the appropriate tarball (let's say "download" or "`curl`" and not "fetch"? :) )
4. Untar it, and explore under `/lib/python3.9/site-packages/` to ensure that `odgi.cpython*.so` is there. If it is elsewhere, make note of the location and substitute in the next step.
5. Add this to your PYTHONPATH: `export PYTHONPATH=...odgi/lib/python3.9/site-packages/`
6. Explore under `/usr/lib/x86_64-linux-gnu/` to ensure that `libjemalloc.so.2` is there. If it is not, search under  `/lib/x86_64-linux-gnu/` and substitute in the next step.
7. Run `export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.2`
8. Run `python` and then `import odgi`.

Reply:
Do you remember which command you used to download the tarball? I tried curl just now but couldn't get it to work for me, so I ended up downloading the tarball to my computer and using `scp`.

Reply:
Ah, this may just be a matter of using the right incantations!
1. `wget internet.address/file.zip` will get `file.zip`, with that name, on your machine.
2. `curl internet.address/file.zip` will give you a silly wall of text, but `curl internet.address/file.zip --output localfile.zip` will get `file.zip`, now renamed `localfile.zip`, on your machine.

FWIW, I was just able to download Odgi for Python 3.9 onto Havarti using both ways. 

Does this help, or is it the case that you already knew this and were facing a deeper issue?

Reply:
Ah, you are correct, I was using the wrong incantation. Thank you for clearing that up!